Sources/GithubAPI.swift

```diff
@@ -407,19 +407,25 @@ public class GithubAPI {
     githubRequestTemplate(requestFlow: performRequest, methodName: #function, resultFlow: nil)
   }
 
-  func getIssueID(issueURL: String) -> Int? {
-    LogFile.debug("getting issue ID with url: \(issueURL)")
-    var issueID: Int?
+  func getObject(objectURL: String) -> [String: Any]? {
```

Comment:
can you please be more verbose in the signature as to what an Object in this case is? Or alternatively some documentation explaining.

Reply:
Done.